package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/hierarchical-token-bucket",
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
Should we just go to 1.0.0? Is the API unstable?

Reply:
maybe later